jupyter_client/tests/test_kernelmanager.py

```diff
@@ -128,10 +128,25 @@ def test_start_new_kernel(self):
         self.assertTrue(km.is_alive())
         self.assertTrue(kc.is_alive())
 
+    def test_kernel_stats(self):
+        c = Config()
+        km = KernelManager(config=c)
+        km.start_kernel(stdout=PIPE, stderr=PIPE)
+        ks = km.kernel_stats
+        # Has to be a dict
+        assert type(ks) is dict
+        # All keys must be a callable function
+        if sys.version_info[:2] == (2, 7):  # version 2.7
+            for _, func in ks.iteritems():
+                assert callable(func)
```

Comment:
For now, I'd assert that the `memory_usage` key exists and that its associated value is an `int`.